Azure/sonic-buildimage
# Minigraph ECMP parsing support (cleaner format)

**- Why I did it**
To support FG_ECMP for certain azd scenarios
**- How I did it**
Modified minigraph parser to parse ECMP fields in the case they are present in minigraph
**- How to verify it**
Loaded ensuing config_db file on a DUT to verify the fields are parsed and configure device correctly
**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
